rebar.config.script

```diff
@@ -0,0 +1,14 @@
+HandleBigEndianPort = fun
+    F([H|T]) when element(1, H) == port_env ->
```

Comment:
@kxepal I am fairly new to Erlang so I probably wasn't writing the smartest code. I have rewritten the script to use lists:keyfind and avoid the fun. Does it look better now?